Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -4593,4 +4593,38 @@ M  END)CTAB";
       CHECK(m->getBondWithIdx(0)->getIsAromatic());
     }
   }
-}
\ No newline at end of file
+}
+
+TEST_CASE(
+    "Github #5165: issue with V3000 SD files containing enhanced "
+    "stereochemistry information") {
+    
+    SECTION("as reported") {
+        std::string rdbase = getenv("RDBASE");
+        std::string fName =
+          rdbase + "/Code/GraphMol/FileParsers/test_data/mol_with_enhanced_stereo_2_And_groups.sdf";
+        SDMolSupplier suppl(fName);
+        ROMol *mol = suppl.next();
+        auto groups = mol->getStereoGroups();
+        CHECK(!groups.empty());
```

Comment:
It's good to make tests specific; since you know what the expected size should be here, you can test that directly:
```suggestion
        CHECK(groups.size()==2);
```